tools/m2-repo-cleaner/src/main/java/org/fusesource/ide/tools/mavenutils/M2RepoCleaner.java

```diff
@@ -120,9 +120,10 @@ private static void handleCorruptedZip(File file) {
 	}
 	
 	private static void handleLastUpdatedFile(File file) {
-		corruptedFiles.add(file.getPath());
-		if (file.isFile() && file.exists()) {
-			file.delete();
-		}
+		handleCorruptedZip(file);
+//		corruptedFiles.add(file.getPath());
```

Comment:
I had 2 runs where I only deleted the lastUpdated files and those builds were red. Once I re-enabled the deletion of the whole folders the build became green again. So I think deleting just the lastUpdated files doesn't work for us.